cli/cli/cli.go

```diff
@@ -278,23 +278,23 @@ func (curveadm *CurveAdm) ClusterTopologyData() string       { return curveadm.c
 func (curveadm *CurveAdm) ClusterPoolData() string           { return curveadm.clusterPoolData }
 func (curveadm *CurveAdm) Monitor() storage.Monitor          { return curveadm.monitor }
 
-func (curveadm *CurveAdm) GetHost(host string) (*hosts.HostConfig, error) {
+func (curveadm *CurveAdm) GetHost(name string) (*hosts.HostConfig, error) {
 	if len(curveadm.Hosts()) == 0 {
 		return nil, errno.ERR_HOST_NOT_FOUND.
-			F("host: %s", host)
+			F("host: %s", name)
 	}
 	hcs, err := hosts.ParseHosts(curveadm.Hosts())
 	if err != nil {
 		return nil, err
 	}
 
 	for _, hc := range hcs {
-		if hc.GetHost() == host {
+		if hc.GetName() == name {
 			return hc, nil
 		}
 	}
 	return nil, errno.ERR_HOST_NOT_FOUND.
-		F("host: %s", host)
+		F("host: %s", name)
```

Comment:
Does the host printed here need to be replaced？

Reply:
The parameter passed in by this method is the name string in the hosts module. Although in other modules, Host is directly used as the identifier, I think the original parameter should be modified in the name in this method. I guess this can make the code The functions in this module are easier to understand
If my understanding is wrong, I would like further guidance on modifying the code there. I will modify the code as soon as possible.